src/main/java/seedu/duke/atomichabit/atomichabit/AtomicHabit.java

```diff
@@ -0,0 +1,26 @@
+package seedu.duke.atomichabit.atomichabit;
+
+public class AtomicHabit {
+    protected String description;
+
+    protected int count;
```

Comment:
Hmm two things here:
1) `private` access is sufficient since you have `protected` getter methods for these variables alr
2) there's an extra newline between `description` and `count`